environment.yml

```diff
@@ -2,16 +2,8 @@ name: polyglotdb-dev
 channels:
     - conda-forge
 dependencies:
-    - librosa
-    - scipy
-    - praatio ~= 5.0
-    - textgrid
-    - influxdb
-    - tqdm
-    - future
-    - requests
-    - openjdk=11
+    - python
+    - openjdk=21
     - pip
     - pip:
-        - conch_sounds
-        - neo4j-driver ~= 4.3
+        - -e .
```

Comment:
Slight nit, I would move the actual installation of polyglot to a separate command wherever it's needed and have environment.yml as just the prerequisites.